orca-clouddriver/src/main/groovy/com/netflix/spinnaker/orca/kato/pipeline/UpdateSecurityGroupsStage.groovy

```diff
@@ -0,0 +1,45 @@
+/*
+ * Copyright 2016 Netflix, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License")
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.netflix.spinnaker.orca.kato.pipeline
+
+import com.netflix.spinnaker.orca.clouddriver.tasks.instance.UpdateInstancesTask
+import com.netflix.spinnaker.orca.clouddriver.tasks.servergroup.ServerGroupCacheForceRefreshTask
+import com.netflix.spinnaker.orca.clouddriver.tasks.servergroup.UpdateLaunchConfigTask
+import com.netflix.spinnaker.orca.pipeline.LinearStage
+import com.netflix.spinnaker.orca.pipeline.model.Stage
+import groovy.util.logging.Slf4j
+import org.springframework.batch.core.Step
+import org.springframework.stereotype.Component
+
+@Slf4j
+@Component
+class UpdateSecurityGroupsStage extends LinearStage {
```

Comment:
I'm not sure this stage name really matches what it's doing.

`UpdateAsgSecurityGroupsStage`?